cicecore/cicedynB/dynamics/ice_dyn_evp.F90

```diff
@@ -1517,19 +1540,19 @@ subroutine stress_U   (nx_block,   ny_block,  &
       ! viscous coefficients and replacement pressure at T point
       !-----------------------------------------------------------------
 
-! COMING SOON!!!
-  zetax2U = c0
-  etax2U = c0
-  rep_prsU = c0
-!         call viscous_coeffs_and_rep_pressure_U (zetax2T(i,j),    zetax2T(i,j+1), &
-!                                                 zetax2T(i+1,j+1),zetax2T(i+1,j), &
-!                                                 etax2T(i,j),     etax2T(i,j+1),  &
-!                                                 etax2T(i+1,j+1), etax2T(i+1,j),  &
-!                                                 hm(i,j),         hm(i,j+1),      &
-!                                                 hm(i+1,j+1),     hm(i+1,j),      &
-!                                                 tarea(i,j),      tarea(i,j+1),   &
-!                                                 tarea(i+1,j+1),  tarea(i+1,j),   &
-!                                                 DeltaU                          )
+         zetax2U = c0
+         etax2U = c0
+         rep_prsU = c0
+         
+         call viscous_coeffs_and_rep_pressure_T2U (zetax2T(i  ,j  ), zetax2T(i  ,j+1), &
+                                                   zetax2T(i+1,j+1), zetax2T(i+1,j  ), &
+                                                   etax2T (i  ,j  ), etax2T (i  ,j+1), &
+                                                   etax2T (i+1,j+1), etax2T (i+1,j  ), &
```

Comment:
This is an example how I prefer to align code by each element (parentheses, i's, j's, commas, &'s).  The last line of this argument list still needs to be aligned.